src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
```

Comment:
The font-weight is set to 500 for the entire body. Ensure that this is the desired weight for all text elements, as it might not be suitable for all text styles, such as headings or emphasized text.